### DISCUSSION TITLE
Plex Friends & Sharing

### DISCUSSION THREAD

**Discussion by lotte67890** [croneter/PlexKodiConnect#1357] [Q&A]
Hello,  I want to use Plex for Kodi (18.9) running on Libreelec (9.2.6) so that I can use my friends Plex Library over the Internet. In the native Plex for Windows app there is a function to add a friend a use his library. Is it possible to have this functionality with PlexKodiConnect? Or somehow else?

Thanks in advance
L

**Comment by ziwzih** [comment]
I have the same question!

**Reply by croneter** [reply]
Use the Plex Media Server webinterface "Plex Web": https://support.plex.tv/articles/201105738-creating-and-managing-server-shares/

**Reply by lotte67890** [reply]
You´re my hero, It works without PlexKodiConnect, Plex for Kodi is enough, but obviously I had two Plex accounts, and the other one was connectet with my friends Library... So thanks for enlightning me!
